src/Services/SwaggerService.php

```diff
@@ -365,25 +366,52 @@ protected function savePostRequestParameters($actionName, $rules, array $annotat
                 ];
             }
 
-            $this->saveDefinitions($actionName, $rules, $annotations);
+            $this->saveDefinitions($actionName, $rules, $attributes, $annotations);
         }
     }
 
-    protected function saveDefinitions($objectName, $rules, array $annotations)
+    protected function saveDefinitions($objectName, $rules, $attributes, array $annotations)
     {
         $data = [
             'type' => 'object',
             'properties' => []
         ];
+
+        $uselessRules = [
```

Comment:
let's make a protected field with the fields type map and use it here and inside the `getParameterType` method